src/pages/Admin/Charity/EditProfile/Form.tsx

```diff
@@ -126,6 +126,13 @@ export default function Form() {
         />
 
         <HideBGTipCheckbox />
+
+        <Field<FV>
+          classes="field-admin"
+          name="slug"
+          label="Slug"
```

Comment:
@SovereignAndrey @ap-justin we should add a bit more detail about the purpose of "Slug", as not all users know what it even is.

Reply:
chose to defer the verbiage to marketing - I should have put this in some comment - Unless you really want this in this PR @SovereignAndrey

Reply:
@ap-justin I would agree we can change that a bit to be more understandable(at least an attempt at it) within this PR. This was my bad. I forgot to add the UI facing wording for that field/help text. :thinking: 

For the UI facing text, let's use the following for now. Marketing can always chime in with teaks, if needed, to the copy:
- [ ] Field Label: `Profile "Vanity" URL`
- [ ] Help Tex below label: `Set your nonprofit's profile URL ending segment from the default numeric ID to a more human-readable value for better SEO and donor recognition. <strong>Only numbers and letters are permitted!</strong> Your numeric ID URL will continue to work still, so there is no risk of breaking existing links that have been shared.</br>Example: Setting this field as "myNonprofit" would change your nonprofit's profile URL to {{baseURL}}/marketplace/myNonprofit.`

Reply:
@SovereignAndrey, AWS were setup for 
```
{{baseURL}}/marketplace?slug=myNonProfit
```
should I change this so that slug is treated as path parameter instead of search param?

Reply:
@ap-justin I think it's fine if the web app takes either, in the cleaner /marketplace/{idOrSlug} setup. :+1: Web App can check if `slugOrId` is an integer. If we have an integer, use the ID lookup method AWS, else use the slug query method: `parseInt(slugOrId) ? endowments/{slugOrId} : endowments?slug={slugOrId}`

Reply:
@SovereignAndrey 
> I think it's fine if the web app takes either, 

just to confim: we should allow web-app urls: 
```
marketplace?slug={slug}
marketplace/id
marketplace/slug
```

and in the field description 
`...would create a new vanity profile URL of {{baseURL}}/marketplace/myNonprofit.`

Reply:
@ap-justin I don't think there's a need for the first option: `marketplace?slug={slug}`.
Description looks OK.

Reply:
> @ap-justin I don't think there's a need for the first option: `marketplace?slug={slug}`. Description URL looks OK.

removed route `marketplace?slug={slug}`